.github/workflows/generate-sandboxes-next.yml

```diff
@@ -48,3 +48,10 @@ jobs:
       - name: Publish
         run: yarn publish-sandboxes --remote=https://storybook-bot:${{ secrets.PAT_STORYBOOK_BOT}}@github.com/storybookjs/sandboxes.git --push --branch=next
         working-directory: ./code
+      - name: The job has failed
+        if: ${{ failure() || cancelled() }}
```

Comment:
pro tip (but also very useless) @ndelangen, you don't need to wrap it in an `${{ expression }}` syntax when writing `if:` declarations, that is automatically handled. So you could do:

```yaml
if: failure() || cancelled()
```